update readme with deployment fixes

acravenho opened the pull request:
Fixes #721 

## Motivation

While creating a fresh deployment yesterday I noticed that the recent `dev.secret.exs` update requiring a user to configure their DB caused errors during deployment. 

## Changelog

- commented out `username` and `password` from `dev.secret.exs.example`. 
- added specific documentation in the README for Linux users to update their DB config
- added some documentation for deploying other chains

### Enhancements
Fix documentation for deploying a new instance.

### Bug Fixes
n/a

### Incompatible Changes
n/a

## Upgrading

n/a

Alaradia commented:
in general you shouldn't have been copying the example without changing it that's why its an example

acravenho commented:
@Alaradia I think you're 100% correct. I've added the fields back in and updated the README. It's important to receive community feedback on where there are issues deploying. We have an issue coming up which will provide a single configuration file for all changes to be made when deploying a chain.

Alaradia commented:
> @Alaradia I think you're 100% correct. I've added the fields back in and updated the README. It's important to receive community feedback on where there are issues deploying. We have an issue coming up which will provide a single configuration file for all changes to be made when deploying a chain.

i do apologise though i felt i came off a bit rude.

acravenho commented:
> > @Alaradia I think you're 100% correct. I've added the fields back in and updated the README. It's important to receive community feedback on where there are issues deploying. We have an issue coming up which will provide a single configuration file for all changes to be made when deploying a chain.
> 
> i do apologise though i felt i came off a bit rude.

Not at all. It's great to have people provide their opinions and support 😄
